batch/batch/cloud/azure/driver/resource_manager.py

```diff
@@ -172,8 +172,7 @@ async def create_vm(
             self.app['feature_flags'],
         )
 
-        memory_mib = azure_worker_memory_per_core_mib(parts.family) * cores
-        memory_in_bytes = memory_mib << 20
+        cores, memory_in_bytes = azure_machine_type_to_cores_and_memory_bytes(machine_type)
```

Comment:
There's already a call to get the `MachineTypeParts` earlier in this function, so you can use that to grab the memory in bytes and delete this function call altogether.